### PR TITLE
Peer20 271

### PR DESCRIPTION
**Pull request by DY-JS**
PEER20-271  Remove "Promote Question" from New Post page

### CODE REVIEW

File: app/components/QuestionForm/TypeForm.jsx [hunk 1]
```diff
@@ -1,29 +1,53 @@
...
-    [formValues],
+    [locale, formValues]
   );
 
+  useEffect(() => {
+    if (descriptionListLabel && descriptionListItems) {
+      setIsType(true);
+      setIsError(false);
+    }
+  }, [descriptionListLabel, descriptionListItems, isError]);
```
Comment (ruslanzharkov) [inline app/components/QuestionForm/TypeForm.jsx:49]: isError is unnecessary in deps array, isn't it?

File: app/components/QuestionForm/index.js [hunk 2]
```diff
@@ -121,11 +115,19 @@ export const QuestionForm = ({
...
   disableCommForm,
 }) => {
+ const [isSelectedType, setIsSelectedType] = useState(false);
+ const [isError, setIsError] = useState(false);
+
   const handleSubmitWithType = sendQuestion => {
     if (communityQuestionsType !== ANY_TYPE) {
       change(FORM_TYPE, communityQuestionsType);
+    } 
+    if(!isSelectedType && !isError){
```
Comment (ruslanzharkov) [inline app/components/QuestionForm/index.js:125]: please use code formatting here with prettier

File: app/components/QuestionForm/index.js [hunk 3]
```diff
@@ -121,11 +115,19 @@ export const QuestionForm = ({
...
+      setIsError(true);
```
Comment (ruslanzharkov) [inline app/components/QuestionForm/index.js:126]: return setIsError(true);

File: app/components/QuestionForm/index.js [hunk 4]
```diff
@@ -121,11 +115,19 @@ export const QuestionForm = ({
...
     }
-    return handleSubmit(sendQuestion);
+    if(!isError && isSelectedType){
+        return handleSubmit(sendQuestion);
```
Comment (ruslanzharkov) [inline app/components/QuestionForm/index.js:129]: and here just return `handleSubmit(sendQuestion);`
without if block

File: app/components/QuestionForm/index.js [hunk 5]
```diff
@@ -163,7 +165,9 @@ export const QuestionForm = ({
 
       <TipsBase>
         <BaseSpecialOne>
-          <FormBox onSubmit={handleSubmitWithType(sendQuestion)}>
+          <FormBox onSubmit={(e)=>{
```
Comment (ruslanzharkov) [inline app/components/QuestionForm/index.js:168]: please not use inline function, create  function for this

File: app/containers/CreateCommunity/QuestionsTypeForm.jsx [hunk 6]
```diff
@@ -43,6 +43,8 @@ const QuestionsTypeForm = ({
     },
     [formValues],
   );
+
+  console.log("f ",formValues);
```
Comment (ruslanzharkov) [inline app/containers/CreateCommunity/QuestionsTypeForm.jsx:47]: remove

File: app/containers/DataCacheProvider/saga.js [hunk 7]
```diff
@@ -48,6 +48,7 @@ import {
 } from './constants';
 import { selectEthereum } from '../EthereumProvider/selectors';
```
Comment (ruslanzharkov) [inline app/containers/DataCacheProvider/saga.js:49]: absolute path

File: app/components/QuestionForm/TypeForm.jsx (hunk 1 above)
Comment (DY-JS) [reply app/components/QuestionForm/TypeForm.jsx:49]: Removed it

File: app/components/QuestionForm/index.js (hunk 3 above)
Comment (DY-JS) [reply app/components/QuestionForm/index.js:126]: agree

File: app/components/QuestionForm/index.js (hunk 4 above)
Comment (DY-JS) [reply app/components/QuestionForm/index.js:129]: isError is nessesery

File: app/components/QuestionForm/index.js (hunk 5 above)
Comment (DY-JS) [reply app/components/QuestionForm/index.js]: chenged

File: app/containers/DataCacheProvider/saga.js (hunk 7 above)
Comment (DY-JS) [reply app/containers/DataCacheProvider/saga.js:49]: What I have  write?

**Pull request closed without merging**